ci-operator/step-registry/redhat-appstudio/load-test/max-concurrency/redhat-appstudio-load-test-max-concurrency-commands.sh

```diff
@@ -99,11 +99,11 @@ if [ "$JOB_TYPE" == "presubmit" ] && [[ "$JOB_NAME" != rehearse-* ]]; then
 fi
 set +x
 
+# Collect load test results at the end
+trap './tests/load-tests/ci-scripts/max-concurrency/collect-results.sh "$SCENARIO"; trap EXIT' SIGINT EXIT
```

Comment:
Hello. What is that `; trap EXIT` for?

Reply:
To avoid duplicating the trap command for multiple signals which would lead to running the trapped command twice - See https://unix.stackexchange.com/a/441598